app/routes/__frontend/resources/virtual-coffee-handbook/join-virtual-coffee/index.mdx

```diff
@@ -0,0 +1,117 @@
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Thank you for being so interested in joining Virtual Coffee! In this article, we're answering the most frequent questions about joining Virtual Coffee!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## How can I join Virtual Coffee?
+
+### 1. Join the waiting list or receive an invite
+
+<LeadText>
+
+We are now accepting new members on a limited basis!
+
+</LeadText>
+
+If you are interested in joining Virtual Coffee, you can <Link to="/join">join the waiting list</Link> or receive an invite from a current member. Once we've accepted and added you to our membership, you will receive an email with links to our Coffees.
+
+### 2. Attend one of our Coffees (Virtual Coffee weekly Zoom chats)
+
+<LeadText>
+
+You must attend one of our <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee#coffees-virtual-coffee-weekly-zoom-chats">Coffees (Virtual Coffee weekly Zoom chats)</Link> on **Tuesday at 9:00 AM ET** or **Thursday at 12:00 PM ET**, which lasts an hour.
+
+</LeadText>
+
+We absolutely love the closeness of this community and know it’s one of the many things that sets us apart. This closeness starts with those Coffees. We would also like to give you the experience of Virtual Coffee in person before we move to the Slack group.
+
+When joining your first Coffee, tell your <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#room-leader">Room Leader</Link> you're a new member. After the Coffee, you'll receive an email with a link to our Slack group and more information.
+
+### 3. Now you’re a full member! Welcome! 👋
+
+You can introduce yourself in the `#welcome` channel and join our members-only events. We would love to know who you are, where you are in your tech journey, and what you enjoy outside of tech!
+
+---
+
+## What is going on in Coffees?
+
+We’re getting to know other members, old and new, through casual conversations on Zoom. We prioritize empathy, encouraging everyone to participate in the most comfortable ways and learning together. Check out the <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee#coffees-virtual-coffee-weekly-zoom-chats">Coffees (Virtual Coffee Weekly Zoom Chats)</Link> section for more details.
+
+---
+
+## Are there any other days and times for Coffee?
+
+Unfortunately, there isn't.
+
+---
+
+## I’m an introvert. Would it be okay to turn off my camera during Coffee?
+
+If you aren't comfortable having your camera or sound on, you can always turn them off. There is no obligation for you to have them on. You can always drop chat if it's more convenient for you. We want you to be as comfortable as possible in our Coffees.
```

Comment:
If you aren't comfortable having your camera or sound on, you can always turn them off. You can always use the chat feature. We want you to be as comfortable as possible in our Coffees.